oscar.py

```diff
@@ -1537,132 +1548,116 @@ class Time_commit_info(Clickhouse_DB):
         | comment | String          |
         | content | String          |
-    '''
+    """
     columns = ['sha1', 'time', 'tree', 'author', 'parent', 'comment', 'content']
 
     def __init__(self, tb_name='commits_all', db_host='localhost'):
         super(Time_commit_info, self).__init__(tb_name, db_host)
     
     def commit_counts(self, start, end=None):
-        ''' return the count of commits between given date and time
+        """ return the count of commits between given date and time
         >>> t = Time_commit_info()
         >>> t.commit_counts(1568656268)
         8
-        '''
-        rows = self.query_select('count(*)', self.tb_name, start, end)
-        return rows[0][0]
+        """
+        return self.query_select('count(*)', self.tb_name, start, end)[0][0]
     
-    def commits_iter(self, start, end=None):
-        ''' return a generator of Commit instances within a given date and time
+    def commits(self, start, end=None):
+        """ return a generator of Commit instances within a given date and time
         >>> t = Time_commit_info()
         >>> commits = t.commits_iter(1568656268)
         >>> c = commits.next()
         >>> type(c)
         <class 'oscar.Commit'>
         >>> c.parent_shas
         ('9c4cc4f6f8040ed98388c7dedeb683469f7210f5',)
-        '''
-        row_iter = self.query_select_iter('lower(hex(sha1))', self.tb_name, start, end)
-        for row in row_iter:
-            yield Commit(row[0])
+        """
+        for sha in self.commits_shas(start, end):
+            yield Commit(sha)
 
     def commits_shas(self, start, end=None):
-        ''' return a list of shas within the given time and date
-        >>> t = Time_commit_info()
-        >>> shas = t.commits_shas(1568656268)
-        >>> type(shas)
-        <type 'list'>
-        '''
-        rows = self.query_select('lower(hex(sha1))', self.tb_name, start, end)
-        return [row[0] for row in rows]
-
-    def commits_shas_iter(self, start, end=None):
-        ''' return a generator of all sha1 within the given time and date
+        """ return a generator of all sha1 within the given time and date
         >>> t = Time_commit_info()
-        >>> for sha1 in t.commits_shas_iter(1568656268):
+        >>> for sha1 in t.commits_shas(1568656268):
         ...     print(sha1)
-        ''' 
-        row_iter = self.query_select_iter('lower(hex(sha1))', self.tb_name, start, end)
-        for row in row_iter:
+        """
+        for row in self.query_select_iter(
+                'lower(hex(sha1))', self.tb_name, start, end):
             yield row[0]
-        
+
+
 class Time_project_info(Clickhouse_DB):
-    ''' Time_project_info class is initialized with table name and database host name
-        The default table name for projects is projects_all, and the default database name is localhost
-        This class contains methods to query for project data
-        The 'b2cPtaPkgR_all' table descrption is the following:
```

Comment:
The table name 'b2cPtaPkgR_all' is the new table, 'projects_all' table has been removed.